### PR TITLE
Update README.md

### LINKED ISSUE #8: Set up README.md to explain how to run the scripts
- prerequisites
- commands
- test mode

### PR DESCRIPTION
**Pull request by srirambv**
Closes #8

### CODE REVIEW

**Comment by luixxiul** [comment]
would you mind renaming the file to `README.md` to get it shown on the top page? thanks!

**Comment by srirambv** [comment]
Updated

File: README.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+#License MPL 2.0
+
+
```
Comment (luixxiul) [inline README.md]: Please remove above this line (we linked MPL 2.0 licence from the top page), thanks!

**Comment by srirambv** [comment]
Done

**Pull request merged**